ui/src/components/flags/FlagTable.tsx

```diff
@@ -33,13 +33,12 @@ export default function FlagTable(props: FlagTableProps) {
 
   const path = `/namespaces/${namespace.key}/flags`;
 
-  const pageSize = 20;
   const searchThreshold = 10;
 
   const [sorting, setSorting] = useState<SortingState>([]);
   const [pagination, setPagination] = useState<PaginationState>({
     pageIndex: 0,
-    pageSize
+    pageSize: 20
```

Comment:
i think this is fine if the minimum page size is 20, however if not its kind of strange that users have to wait for 20 rows until the pagination shows up, then they can set it to 10

Reply:
It was just a work in progress. So 20 is the minimal.